internal/controller/helpers.go

```diff
@@ -1,155 +0,0 @@
-// Copyright Project Contour Authors
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package controller
```

Comment:
Has this file been removed because we're not using it any more? How are the conditions for Gateway etc being set again?

Reply:
Yeah, not using it anymore -- I created this file when I added the TCPRoute/UDPRoute controllers to share some logic between them. Logic for setting GatewayClass/Gateway conditions are in the respective controllers, or in the DAG.